### PR TITLE
[12.0][MIG] website_menu_by_user_status

### PR DESCRIPTION
**Pull request by smurf-U**

### CODE REVIEW

**Comment by nikul-serpentcs** [comment]
@smurf-U In Readme, use code-gen
readme like this https://github.com/OCA/maintainer-tools/tree/master/template/module/readme

File: website_google_tag_manager/README.rst
Comment (gdgellatly) [inline website_google_tag_manager/README.rst]: Why is this file here?

**Review by gdgellatly — COMMENTED** [review]
Otherwise LGTM

File: website_google_tag_manager/README.rst
Comment (smurf-U) [reply website_google_tag_manager/README.rst]: @gdgellatly Nothings...
But because of that Travis failed.
After change it done

Comment (gdgellatly) [reply website_google_tag_manager/README.rst]: Doesn't matter I think. It should go in a separate PR. It can only have been a lint error. In those cases just comment in the PR and say Travis failed because of unrelated module. 

But I let @pedrobaeza say, I don't know every arcane rule of OCA PR's and changes to lint check etc.

Comment (pedrobaeza) [reply website_google_tag_manager/README.rst]: @smurf-U I doubt this file as it was have any Travis problem. @gdgellatly there are no new rules nor lint checks. Simply proper RST formatting (and people tend to break it), but as I see, it's not the case. If not, please show Travis log to check.

Comment (smurf-U) [reply website_google_tag_manager/README.rst]: @pedrobaeza You can refer below like for Travis failed log
https://travis-ci.org/OCA/website/builds/476770521

Which Give error-

> ************* Module website_google_tag_manager
> website_google_tag_manager/README.rst:3: [E7901(rst-syntax-error), ]  Duplicate implicit target name: "google tag manager".
> ************* Module website_menu_by_user_status

Comment (gdgellatly) [reply website_google_tag_manager/README.rst]: @smurf-U Feel free to make a separate PR but that error will not stop this PR merging, it is in an unrelated module.

Tidying up these ones however would be nice.

************* Module website_menu_by_user_status
website_menu_by_user_status/README.rst:1: [W7908(missing-newline-extrafiles), ]  Missing newline
************* Module website_menu_by_user_status.models.website_menu
website_menu_by_user_status/models/website_menu.py:17: [W8103(translation-field), WebsiteMenu] Translation method _("string") in fields is not necessary.
website_menu_by_user_status/models/website_menu.py:25: [W8103(translation-field), WebsiteMenu] Translation method _("string") in fields is not necessary.
website_menu_by_user_status/models/website_menu.py:31: [W8104(api-one-deprecated), WebsiteMenu._compute_visible] api.one deprecated

Comment (gdgellatly) [reply website_google_tag_manager/README.rst]: In any case, this is already fixed here https://github.com/OCA/website/pull/576/files so rebasing PR on HEAD will resolve for you.

Comment (smurf-U) [reply website_google_tag_manager/README.rst]: @gdgellatly will rebase...

File: website_menu_by_user_status/models/website_menu.py
Comment (flotho) [inline website_menu_by_user_status/models/website_menu.py]: Hi, I would suggest to change the string by : 
``` Visible for logged Users```

File: website_menu_by_user_status/models/website_menu.py
Comment (flotho) [inline website_menu_by_user_status/models/website_menu.py]: Hi, I would suggest to change the string by : 
``` Visible for public Users```

**Review by flotho — APPROVED** [review]
small comments from my side, otherwise, LGTM

**Comment by simahawk** [comment]
any chance to see this based on https://github.com/OCA/website/tree/11.0/website_menu_permission ?
Basically the 2 flags will just toggle specific groups automatically.
Otherwise we can plan this for v13.

**Comment by smurf-U** [comment]
> any chance to see this based on https://github.com/OCA/website/tree/11.0/website_menu_permission ?
> Basically the 2 flags will just toggle specific groups automatically.
> Otherwise we can plan this for v13.

It's good to plan this for v13.

**Comment by flotho** [comment]
Hi @simahawk ,

This could be a great idea, yet name of the module won't respect the main usage and it will add more time to get a full review of this module.
Maybe we could do this in a next step once it has been merged ?
What's your point of view on this ?

Regards

**Review by flotho — APPROVED** [review]

**Comment by simahawk** [comment]
not a blocker, just add it to the ROADMAP pls

**Comment by flotho** [comment]
Hi @gdgellatly 

As you started a review, any chance you approve this PR ?

**Comment by flotho** [comment]
hi @smurf-U 

any chance to take comment https://github.com/OCA/website/pull/559#issuecomment-546882167 into account ?

**Comment by smurf-U** [comment]
> hi @smurf-U
> 
> any chance to take comment [#559 (comment)](https://github.com/OCA/website/pull/559#issuecomment-546882167) into account ?

@flotho  Done

**Comment by flotho** [comment]
@simahawk , @gdgellatly 

Any chance to approve this PR so that @pedrobaeza will be able to merge.

regards

**Review by simahawk — APPROVED** [review]

**Review by simahawk — CHANGES_REQUESTED** [review]
https://github.com/OCA/website/pull/559/files#diff-03b9c0f284e6f2c8c069276b08065d5f this change is wrong

**Comment by flotho** [comment]
@smurf-U , we're almost done, a last comment with bad reference into readme.

**Comment by smurf-U** [comment]
> a last comment with bad reference into readme.

I'm not getting you.

**Comment by flotho** [comment]
@smurf-U I think the point is that you're refering a readme of another module there : https://github.com/OCA/website/pull/559/files#diff-03b9c0f284e6f2c8c069276b08065d5f
this is the reason of this comment : https://github.com/OCA/website/pull/559#pullrequestreview-339159776

**Comment by flotho** [comment]
ping @smurf-U 
Easy to change.

**Comment by rafaelbn** [comment]
Is ther any interest in this PR? Thanks

**Comment by smurf-U** [comment]
Apologize will update by ASAP

**Comment by smurf-U** [comment]
> Is ther any interest in this PR? Thanks

@rafaelbn Done

@simahawk can you please review again

**Review by claudiagn — COMMENTED** [review]
@simahawk which changes should be made? in order to also do a functional review and not to miss what you want to change! thx

**Review by simahawk — APPROVED** [review]

**Comment by simahawk** [comment]
Anything else to be done here? Otherwise let's merge.
@dreispt ?

**Pull request merged**
